e2e/poms/submissions.ts

```diff
@@ -0,0 +1,31 @@
+import { Page, expect } from "@playwright/test";
+import Dashboard from "./dashboard";
+import { SUBMISSIONS_SELECTORS, NAVIGATION_SELECTORS } from "../constants/selectors";
+import { DASHBOARD_LINKS } from "../constants/links/dashboard";
+
+export default class SubmissionsPage {
+    constructor(private page: Page) {
+        this.page = page;
+    }
+
+    searchSubmission = async (email: string) => {
+        await this.page.getByTestId(SUBMISSIONS_SELECTORS.searchTextField).fill(email);
+        await expect(this.page.getByTestId(SUBMISSIONS_SELECTORS.searchTermBlock)).toBeVisible();
+    }
+
+    verifySubmission = async (email: string) => {
+        await this.searchSubmission(email);
+        const submissionCountText = await this.page.getByTestId(SUBMISSIONS_SELECTORS.countLabel).innerText();
+        let countNumber = 0;
+        if (submissionCountText && submissionCountText.trim().length > 0) {
+            countNumber = parseInt(submissionCountText.split(' ')[0]) || 0;
+        }
+        expect(countNumber).toBe(1);
```

Comment:
```suggestion
        const submissionCountText = await expect(this.page.getByTestId(SUBMISSIONS_SELECTORS.countLabel)).toHaveText("1 Submission")
```

This is a much easier and reliable assertion. Make sure to move `1 Submission` to texts